src/tests/ftest/util/nvme_utils.py

```diff
@@ -155,7 +155,7 @@ def create_container(self):
 
         self.nvme_local_cont.create()
 
-    def start_ior_thread(self, create_cont, operation):
+    def start_ior_thread(self, create_cont, operation, repetitions=1):
```

Comment:
In addition to this change we also need to update the `def start_ior_load(self, storage='NVMe', operation="WriteRead", percent=1, create_cont=True):` method to add an optional `repetitions=1` argument which it can then pass to this method.

Reply:
ok